cirq-core/cirq/qis/clifford_tableau.py

```diff
@@ -583,6 +583,6 @@ def apply_global_phase(self, coefficient: linear_dict.Scalar):
         pass
 
     def measure(
-        self, axes: Sequence[int], seed: 'cirq.RANDOM_STATE_OR_SEED_LIKE' = None
+        self, axes: Sequence[int], seed: 'cirq.RANDOM_STATE_OR_SEED_LIKE' = np.random
     ) -> List[int]:
```

Comment:
I think I would keep this None and then do a `seed or np.random` in the parameter in `self._measure`.   That way we don't have to change the default here (though I don't think it will matter much).  `np.random` might be considered mutable(?) since it changes when different people use it, so best to not put mutables as defaults.

Reply:
Do we want it as a parameter or like:

```python

    def _measure(self, q, prng: Optional[np.random.RandomState]) -> int:
        if prng is None:
            prng = np.random
```

Reply:
that could also work, though this also does something I try to avoid which is re-assign a parameter.  Mostly this doesn't ever hurt you, but there are cases where it does (mostly having to do with pytest, btw).  It also sort of makes sense that the private method has a random state, but the public one has the default of None.

Reply:
whichever way you want 👍

Reply:
Ah, makes sense. Switched to param.